.github/workflows/python.yaml

```diff
@@ -72,4 +72,11 @@ jobs:
       with:
         version: ${{ env.CURRENT_VERSION }}
         alias: development
-        push: ${{ github.ref == 'refs/heads/main' }}  # build always, publish on 'main' only to prevent version clutter
+        push: ${{ github.ref == 'refs/heads/main' }} # build always, publish on 'main' only to prevent version clutter
+    - name: Execute jupytext in demo venv
```

Comment:
See below for rationale, but let's fold the logic of this step into the `mike-docs` action above.